Probesys/bileto
# Imp/misc

Changes proposed in this pull request: see commits

How to test the feature manually:

1. reset the database
2. open the "organizations" page, check it shows an improved placeholder

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
